source/background/library/archives.js

```diff
@@ -137,16 +130,16 @@ export function addMyButtercupArchive(payload) {
             refreshToken,
             clientID: MYBUTTERCUP_CLIENT_ID,
             clientSecret: MYBUTTERCUP_CLIENT_SECRET,
-            vaultID
+            vaultID,
```

Comment:
Trailing commas could be removed, as per comment on `.prettierrc`

Reply:
Ok (: